src/components/cdPipeline/CDPipeline.tsx

```diff
@@ -1018,7 +1009,7 @@ export default class CDPipeline extends Component<CDPipelineProps, CDPipelineSta
                     </button>
                 )
             } else {
-                return (
+                return this.isWebhookCD? null :(
```

Comment:
Change to `!this.isWebhookCD && (...JSX)`

Reply:
done